.changeset/mean-beds-notice.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux/adp-tooling': patch
+'@sap-ux/preview-middleware': patch
+---
```

Comment:
Why we need patch update on '@sap-ux/preview-middleware' ?

Reply:
Because @sap-ux/preview-middleware is consumes @sap-ux/adp-tooling, otherwise the version in @sap-ux/preview-middleware will not be updated.